compiler/src/dotty/tools/dotc/typer/Docstrings.scala

```diff
@@ -17,13 +17,14 @@ trait Docstrings { self: Typer =>
     */
   def cookComments(syms: List[Symbol], owner: Symbol)(implicit ctx: Context): Unit =
     ctx.docCtx.foreach { docbase =>
-      val relevantSyms = syms.filter(docbase.docstring(_).isDefined)
+      val relevantSyms = syms.filter(docbase.docstring(_).exists(!_.isExpanded))
       relevantSyms.foreach { sym =>
         expandParentDocs(sym)
         val usecases = docbase.docstring(sym).map(_.usecases).getOrElse(Nil)
 
         usecases.foreach { usecase =>
-          enterSymbol(createSymbol(usecase.untpdCode))
+          val createdSymbol = enterSymbol(createSymbol(usecase.untpdCode))
+          createdSymbol.setFlag(Flags.DocArtifact)
```

Comment:
Can you avoid creating a new flag and use the symbol name instead (i.e. is the name a `DocArtifactName`)?

Reply:
I didn't know about `name.is`, thanks!